app/src/pages/PageProjects.re

```diff
@@ -1,22 +1,18 @@
 open Source.Project;
-open Router;
 open Atom;
+open Layout;
+open Molecule;
+
+module Styles = {
+  open Css;
+
+  let projectHeading = style([marginTop(px(94)), marginBottom(px(48))]);
+};
 
 module List = {
   [@react.component]
   let make = (~projects: array(project)) => {
-    let ps =
-      Array.map(
-        project =>
-          <li key={project.address}>
-            <Link page={Project(project.address)}>
-              <Title> {React.string(project.name)} </Title>
-              <p> {React.string(project.description)} </p>
-              <img src={project.imgUrl} />
```

Comment:
As commented above I'd see the `ProjectCard` go inside here, so that it just depends on the three things (`name`, `description`, `imgUrl`). Makes it more dumb and only care about presenting.